WorkoutTest/ProgramViewController.swift

```diff
@@ -56,17 +52,42 @@ final class ProgramViewController: UIViewController {
       self?.myRefreshControl.endRefreshing()
     })
   }
+  
+  private func configureSearchBar() {
+    searchBar = UISearchBar()
```

Comment:
same as for  collectionView = UICollectionView(frame: view.bounds, collectionViewLayout: setupLayout())